README.md

```diff
@@ -170,6 +174,10 @@ Specifies whether to overwrite the destination file if it already exists. Valid
 
 Specifies a validation command to apply to the destination file. Requires Puppet version 3.5 or newer. Valid options: a string to be passed to a file resource. Default value: undefined.
 
+#####`warn`
+
+Deprecated as of concat v2.0.0. Has no effect. Please use `warn_header` instead.
```

Comment:
We've removed warn_header completely. 'warn' was readded to replace warn_header.